src/astro/sql/operators/base_decorator.py

```diff
@@ -12,7 +12,7 @@
 from astro.utils.table import find_first_table
 
 
-class BaseSQLOperator(DecoratedOperator):
+class BaseSQLDecoratorOperator(DecoratedOperator):
```

Comment:
Should it be `BaseSQLDecoratedOperator` instead of "Decorator? Mainly coz it is inheriting `DecoratedOperator`, wdyt?

```suggestion
class BaseSQLDecoratedOperator(DecoratedOperator):
```